app/src/ui/history/compare.tsx

```diff
@@ -401,35 +401,34 @@ export class CompareSidebar extends React.Component<
         event.preventDefault()
         return
       }
-
-      if (this.props.compareState.filterText.length === 0) {
-        this.handleEscape()
+      if (this.state.focusedBranch == null) {
```

Comment:
:art: can you use `===` here instead of `==`? We control the value here, and it should never be `undefined`...

Reply:
Also done!